compliance_checker/suite.py

```diff
@@ -269,11 +269,10 @@ def get_points(self, groups, limit):
         score_list = []
         score_only_list = []
 
-        for v in range(len(groups)):
-            score_list.append([groups[v].name, groups[v].weight, groups[v].value,
-                               groups[v].children])
-            if groups[v].weight >= limit:
-                score_only_list.append(groups[v].value)
+        for g in groups:
```

Comment:
there was probably a reason this was indexed (v was used somehow) but can't remember why.